test/Microsoft.AspNet.Mvc.FunctionalTests/FileResultTests.cs

```diff
@@ -58,7 +58,7 @@ public class FileResultTests
 
             var contentDisposition = response.Content.Headers.ContentDisposition.ToString();
             Assert.NotNull(contentDisposition);
-            Assert.Equal("attachment; filename=downloadName.txt", contentDisposition);
+            Assert.Equal("attachment; filename=downloadName.txt; filename*=UTF-8''downloadName.txt", contentDisposition);
```

Comment:
ContentDispositionHeaderValue.SetHttpFileName intentionally sets both filename and filename* for broadest interop. The values will diverge if the file-name contains non-ascii.